### PR TITLE
Stop reexporting style types from layout_interface.

### PR DESCRIPTION
**Pull request by Ms2ger**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9245)

### CODE REVIEW

File: components/script/layout_interface.rs [hunk 1]
```diff
@@ -24,12 +24,11 @@ use std::any::Any;
 use std::sync::Arc;
 use std::sync::mpsc::{Receiver, Sender, channel};
 use string_cache::Atom;
+use style::context::ReflowGoal;
 use style::stylesheets::Stylesheet;
 use url::Url;
 use util::ipc::OptionalOpaqueIpcSender;
 
-pub use style::animation::Animation;
```
Comment (KiChjang) [inline components/script/layout_interface.rs]: Is this not being used anywhere in this source file at all?

**Comment by nox** [comment]
@KiChjang It isn't.

@bors-servo r+

**Pull request merged**